src/extensions/default/JavaScriptQuickEdit/main.js

```diff
@@ -196,6 +197,14 @@ define(function (require, exports, module) {
             return null;
         }
 
+        //Send analytics data for Quick Edit open
+        HealthLogger.sendAnalyticsData(
```

Comment:
Again do we are just capturing quick edit performed. Are we interested in errors which we get in this operation?

Reply:
we just capture logging when quick edit is performed.